cdap-ui/webpack.config.cdap.js

```diff
@@ -164,6 +166,14 @@ if (mode !== 'production') {
   webpackConfig = Object.assign({}, webpackConfig, {
     devtool: 'source-map',
     plugins:  plugins.concat([
+      new BrowserSyncPlugin({
+          host: 'localhost',
+          port: 3000,
+          proxy: 'http://localhost:11011',
+          browser: ["google chrome", "firefox"]
```

Comment:
We could. Since Chrome and firefox are common across OS platforms I added that for now. Since we will be testing windows predominantly we can test it in safari independently. If we open the same URL in safari too it should just sync our interaction.